builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -33,7 +33,7 @@ func TestAccVSphereVirtualMachine_basic(t *testing.T) {
 	if v := os.Getenv("VSPHERE_DATASTORE"); v != "" {
 		datastoreOpt = fmt.Sprintf("        datastore = \"%s\"\n", v)
 	}
-	template := os.Getenv("VSPHERE_TEMPLATE")
+	cloneSource := os.Getenv("VSPHERE_CLONE_SOURCE")
```

Comment:
The signatures are driving me nuts 😄  LOL.  Definitely a conversation that we can have on slack or another channel.